src/plugins/discover/public/application/embeddable/saved_search_embeddable.tsx

```diff
@@ -286,10 +285,11 @@ export class SavedSearchEmbeddable
     };
 
     const timeRangeSearchSource = searchSource.create();
-    timeRangeSearchSource.setField('filter', () => {
-      if (!this.searchProps || !this.input.timeRange) return;
-      return this.services.timefilter.createFilter(indexPattern, this.input.timeRange);
-    });
+    if (this.searchProps && this.input.timeRange) {
+      timeRangeSearchSource.setField('filter', [
```

Comment:
any consequences of not setting the `filter` field at all in this scenario?

Reply:
ignore this PR for now its been just a test